src/ffi/presentation.rs

```diff
@@ -195,6 +195,50 @@ pub extern "C" fn anoncreds_create_presentation(
     })
 }
 
+/// Optional value for overriding the non-revoked interval in the PresentationRequest
+/// This only overrides the `from` value as a Revocation Status List is deemed valid until the next
+/// entry.
+///
+/// E.g. if the ledger has Revocation Status List at timestamps [0, 100, 200],
+/// let's call them List0, List100, List200. Then:  
+///
+/// ```txt
+///
+///       List0 is valid  List100 is valid
+///        ______|_______ _______|_______
+///       |              |               |
+/// List  0 ----------- 100 ----------- 200
+/// ```
+///
+/// A `nonrevoked_interval = {from: 50, to: 150}` should accept both List0 and
+/// List100.  
+///
+#[derive(Debug)]
+#[repr(C)]
+pub struct FfiNonrevokedIntervalOverride<'a> {
+    rev_reg_def_id: FfiStr<'a>,
+    /// Timestamp in the `PresentationRequest`
+    requested_from_ts: i64,
```

Comment:
Might be good later on to use i32s here. I think we map them to a u64 later and it would help in the JS side where every number is an f32.

Reply:
ah good idea! fixed